LATEST

```diff
@@ -1,3 +1,4 @@
+1f24348e2178fb2dab022ff0cf3d75c5dfa484a6 2.1.0-snapshot.20220308.9472.0.1f24348e
```

Comment:
this should be a split release if at all but as before, you don’t actually need this. the nightly release is good enough.

Reply:
@cocreature I thought the nightly release was the output of `deps list`? Am I supposed to get them from somewhere else?

Reply:
`deps list` lists nightlies that have already been completed, and are therefore eligible for complete split release.